recipes/endstate_correction/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set name = "endstate_correction" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/wiederm/endstate_correction/archive/refs/tags/v0.3.tar.gz
```

The updating bot won't work unless the version and the tag matches.

```suggestion
  url: https://github.com/wiederm/endstate_correction/archive/refs/tags/v{{ version }}.tar.gz
```